rtrlib/config.h.cmake

```diff
@@ -0,0 +1,24 @@
+/*
+ * This file is part of RTRlib.
+ *
+ * This file is subject to the terms and conditions of the MIT license.
+ * See the file LICENSE in the top level directory for more details.
+ *
+ * Website: http://rtrlib.realmv6.org/
+ */
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#ifndef CONFIG_H
```

Comment:
One last thing: this should be `RTR_CONFIG_H`.

Reply:
Piece of cake. Done.